cmake/adjust_global_compile_flags.cmake

```diff
@@ -131,7 +131,11 @@ if (onnxruntime_DISABLE_RTTI)
     # Disable RTTI and turn usage of dynamic_cast and typeid into errors
     add_compile_options("$<$<COMPILE_LANGUAGE:CXX>:/GR->" "$<$<COMPILE_LANGUAGE:CXX>:/we4541>")
   else()
-    add_compile_options("$<$<COMPILE_LANGUAGE:CXX>:-fno-rtti>")
+    # Avoid unboundTypeError for WebNN EP since unbound type names are illegal with RTTI disabled
+    # in Embind API, relevant issue: https://github.com/emscripten-core/emscripten/issues/16911
```

Comment:
should generate a failure if both `--disable_rtti` and `--use_webnn` are specified, instead of silently ignoring the flag.

Reply:
Good point, thanks! Done.